object_tracker/custom_tracker.py

```diff
@@ -3,11 +3,28 @@
 from pytracking.evaluation.multi_object_wrapper import MultiObjectWrapper
 from typing import List, Tuple
 from numpy import ndarray
-
+import numpy as np
 
 class CustomTracker(Tracker):
     def __init__(self, *args, **kwargs):
```

Comment:
Lacks class documentation, please add documentation according to google style format
https://google.github.io/styleguide/pyguide.html